packages/patternfly-4/react-core/src/experimental/components/Drawer/DrawerPanelContent.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/Drawer/drawer';
+import { css } from '@patternfly/react-styles';
+
+export interface DrawerPanelContentProps extends React.HTMLProps<HTMLDivElement> {
+  /** Additional classes added to the Drawer. */
+  className?: string;
+  /** Content to rendered in the drawer */
+  children?: React.ReactNode;
+  /** Indicates if there should be padding around the drawer */
+  hasPadding?: boolean;
+}
+
+export const DrawerPanelContent: React.SFC<DrawerPanelContentProps> = ({
+  className = '',
+  children,
+  hasPadding = true,
+  ...props
+}: DrawerPanelContentProps) => (
+      <aside className={css(styles.drawerPanel, className)} {...props}>
+        <div className={css(styles.drawerPanelBody, hasPadding === false && styles.modifiers.noPadding)}>
```

Comment:
Should we use `noPadding`? We have this same feature in the wizard, and the prop is called `hasBodyPadding`. We also have it in the page component and data list and it's called `noPadding`. We support this feature in core in the table, too, but I looked in the react docs for the table component and couldn't find a reference to it, so not sure if we don't support it, or if we do and it's not documented.

And whichever direction we go, can you add it to the documentation?

Reply:
i'll change it to noPadding.